File: 50 Coding Challenge Part II/TypeScript/Challenge33.ts
// Challenge 33: Calculate the sum of numbers received in a comma delimited string

function sumCSV(num: string): number{

    let ar: string[] = num.split(",");
    let sum: number = 0;

    for (let n in ar){
        sum += parseFloat(ar[n]);
    }

    return sum;
}

// Driver Method
let csvNum: string = "1.5, 2.3, 3.1, 4, 5.5, 6, 7, 8, 9, 10.9";
console.log(sumCSV(csvNum));

File: 50 Coding Challenge Part II/Python/Challenge33.py
# Challenge 33: Calculate the sum of numbers
# received in a comma delimited string


def sumCSV(csvFile):
    sum_csv = 0
    arr = csvFile.split(",")

    for i in arr:
        sum_csv += int(i)

    return sum_csv


# Driver Method
print(sumCSV("32, 24, 45, 38, 5, 1"))
